plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -117,10 +120,12 @@
      *            RegEx patterns of files to exclude from the report
      * @param includePattern
      *            RegEx patterns of files to include in the report
+     * @deprecated This constructor is called internally only, but if you need to use it (for some strange reason), call
+     *            {@link #FindBugsPublisher()} and available setters
      */
     // CHECKSTYLE:OFF
     @SuppressWarnings("PMD.ExcessiveParameterList")
-    @DataBoundConstructor
+    @Deprecated
```

Comment:
Yes, please delete deprecated methods. This makes the plug-in code too polluted...